### PR TITLE
Inline comments

### PR DESCRIPTION
**Pull request by amieiro**
This PR adds inline comments in the discussion tab.

When you click the "Reply" link

image

the web will show you a form below the comment where you can add the reply.

image

If you click the "Cancel Reply", the form hides and the link text changes to "Reply".

image

If you add a comment

image

it will be created and shown as a nested comment.

image

This code doesn't hide another forms when you show a new one.

### CODE REVIEW

File: helpers/helper-translation-discussion.php [hunk 1]
```diff
@@ -328,6 +328,35 @@ function( $link, $args, $comment, $post ) {
...
+			            ),
+		            ),
+		            $comment->comment_post_ID
+	            );
+            } else {
+	            echo sprintf( __( 'You have to be <a href="%s">logged in</a> to comment.' ), wp_login_url() );
+            }
+            ?>
+            </div>
 		</footer>
```
Comment (akirk) [inline helpers/helper-translation-discussion.php:360]: This is the `<footer>` causing the cursive content.

**Review by akirk — CHANGES_REQUESTED** [review]
It works well but all the text is in italics because it is inside a `footer` tag (which has CSS `article.comment { font-size: 0.8rem; font-style: italic; }`, could you please change that?

**Comment by amieiro** [comment]
The nested comment has now normal font style, not italic.

image

File: js/translation-helpers.js [hunk 2]
```diff
@@ -63,6 +64,15 @@ $gp.translation_helpers = (
 				$tab.addClass('current');
 				$("#"+tab_id).addClass('current');
 			},
+			reply_comment_form: function( $comment ) {
+				var commentId = $comment.attr('data-commentid') ;
+				$('#comment-reply-' + commentId).toggle();
+				if ('Reply' === $comment.text()) {
+					$comment.text('Cancel Reply');
+				} else {
+					$comment.text('Reply');
```
Comment (akirk) [inline js/translation-helpers.js:73]: Some small whitespace issues:

```suggestion
				var commentId = $comment.attr( 'data-commentid' );
				$( '#comment-reply-' + commentId ).toggle();
				if ( 'Reply' === $comment.text() ) {
					$comment.text( 'Cancel Reply' );
				} else {
					$comment.text( 'Reply' );
```

File: js/translation-helpers.js [hunk 3]
```diff
@@ -71,8 +81,13 @@ $gp.translation_helpers = (
...
 					$gp.translation_helpers.tab_select( $( this ) );
 					return false;
-				}
-			}
+				},
+                reply_comment_form: function ( event ) {
+                    event.preventDefault();
+					$gp.translation_helpers.reply_comment_form( $( this ) );
+                    return false;
+               },
```
Comment (akirk) [inline js/translation-helpers.js:89]: Mix of tabs and spaces:
```suggestion
				reply_comment_form: function ( event ) {
					event.preventDefault();
					$gp.translation_helpers.reply_comment_form( $( this ) );
					return false;
				},
```

File: helpers/helper-translation-discussion.php [hunk 4]
```diff
@@ -328,6 +328,35 @@ function( $link, $args, $comment, $post ) {
 				<?php $translation = GP::$translation->get( $comment_translation_id ); ?>
 				<em>Translation: <?php echo esc_translation( $translation->translation_0 ); ?></em>
 			<?php endif; ?>
+            <div class="clear"></div>
+            <div id="comment-reply-<?php echo $comment->comment_ID; ?>" style="display: none;">
+            <?php
+            if ( is_user_logged_in() ) {
```
Comment (akirk) [inline helpers/helper-translation-discussion.php:334]: In this section there is also a mix of tabs and spaces.

**Review by akirk — COMMENTED** [review]
Just a few whitespace issues, possibly this will be caught in future with #7.

File: helpers/helper-translation-discussion.php (hunk 1 above)
Comment (amieiro) [reply helpers/helper-translation-discussion.php:360]: Done in the last commits.

**Review by akirk — APPROVED** [review]
Let's go with this!

**Pull request merged**